app/assets/javascripts/spree/frontend/spree_payu_integration.js

```diff
@@ -1,13 +1,10 @@
-/* Placeholder for backend dummy app */
 (function($) {
   SpreePayuIntegration = {
-    updateSaveAndContinueVisibility: function(e) {
+    updatePayuSelectedClass: function(e) {
```

Comment:
til: state param to `toggleClass`, works like a charm @RaVbaker 

merging becase no more issue are left